crates/bench-api/src/lib.rs

```diff
@@ -211,9 +211,22 @@ impl BenchState {
 
         let mut linker = Linker::new(&self.store);
 
-        // Import a very restricted WASI environment.
+        // Create a WASI environment.
+
         let mut cx = WasiCtxBuilder::new();
         cx.inherit_stdio();
+        // Allow access to the current working directory so that the benchmark
+        // can read its input workload(s).
+        cx.preopened_dir(
+            File::open(".").context("failed to open the current working directory")?,
+            ".",
+        );
```

Comment:
Not sure this is correct: if we run the benchmarks from the sightglass root directory, we would preopen that instead of `benchmarks-next/[some benchmark]` (which is what I thought we wanted to do). Unfortunately, to preopen `benchmarks-next/[some benchmark]` we probably need to pass a path in--the path to the Wasm file would work--or alter the current directory before running each benchmark.

Reply:
This is running in `benchmarks-next/$name` because the in-process benchmark command is spawned with the correct current directory set

Reply:
Cool; should we document that here?

Reply:
Added a comment